playerctl/playerctl-cli.c

```diff
@@ -108,7 +110,25 @@ int main (int argc, char *argv[])
 
     if (command[1]) {
       /* set */
-      level = g_ascii_strtod(command[1], NULL);
+
+      if(g_str_has_prefix(command[1], "+") || g_str_has_prefix(command[1], "-")) {
```

Comment:
In case of `playerctl + 0.2` style, I would prefer if it is exactly equal to + instead of just checking the prefix. An error otherwise would be good.